### ISSUE TITLE
Add axe testing to cypress

### ISSUE THREAD

**Issue by georgehager** [CDCgov/prime-simplereport#4530]
## Background

After an initial spike to assess feasibility of adding axe accessibility testing to cypress, see #4529, we should add axe testing to each page in cypress to prevent accessibility regressions.

## Action Requested
Implement axe tests on each page we test with cypress. The test should be for the whole page, not individual components, using something like [`cy.checkA11y()`](https://timdeschryver.dev/blog/setting-up-cypress-with-axe-for-accessibility#configuration) for each page.

If the testing uncovers existing accessibility issues, those should be logged as new tickets, and we should either 1) set up axe to ignore those issues short term until they are resolved with follow-on tickets or 2) make e2e testing non-blocking for PRs until all issues are resolved. Option 2 is not recommended.

## Acceptance Criteria
Each page we test with cypress has a page level axe accessibility test.

## Additional Context
Whenever a new page is added to the app, cypress tests that get added should include the accessibility check.
